packages/vkui/src/components/FocusTrap/FocusTrap.test.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import { act } from 'react';
```

Comment:
```suggestion
import { act } from 'react';
```

недопоняли друг друга)) я имел ввиду прямо тут

Reply:
`React.*` используется же в файле, убирать не надо

Reply:
ой, пардонте...

```diff
- import * as React from 'react';
- import { act } from 'react';
+ import { act, Fragment, useRef, useState } from 'react';
```

Reply:
Остановитесь) 
Может оставим везде `* as React`? Мы и так это везде используем, `act` не так уж часто экспортируется в тестах, пусть будет `React.act`.

Reply:
@inomdzhon Поправил импорты

Reply:
> Остановитесь) Может оставим везде `* as React`? Мы и так это везде используем, `act` не так уж часто экспортируется в тестах, пусть будет `React.act`.

в тест файлах сейчас именованные экспорты